### DISCUSSION TITLE
Pb param Upload media size

### DISCUSSION THREAD

**Discussion by Mika17420** [joomla/joomla-cms#43089] [General]
I try to transfer a PDF file to the image folder of my Joomla 4. But it keeps telling me that the file is too large.
My php settings are good at 8MB. The media/config setting is 4MB. But as soon as I send my pdf which is 1.7 MB it systematically reduces the media settings to 2 MB.
I have several sites and the problem is the same everywhere
an idea ?

**Comment by dautrich** [comment]
Did you increase at least PHP settings post_max_size and upload_max_filesize ?

**Reply by Mika17420** [reply]
yes I made this setting.
on the other hand in joomla media param, if I put 0. The value is retained and it blocks according to the size indicated in php upload_max_file.

**Comment by Mika17420** [comment]
false alarm
after refreshing the page, the value dropped again to 2
